mail-integration-webapp/src/main/resources/locale/portlet/MailIntegration_en.properties

```
@@ -31,3 +31,7 @@ mailIntegration.settings.connectMail.errorMessage=Saving mail settings failed
 UINotification.label.mailIntegration=Email integration Notification
 UINotification.label.MailReceived=Email received
 UINotification.title.MailIntegrationNotificationPlugin=Email received
+UINotification.MailIntegrationNotification.drawer.title=Mails
+UINotification.message.received.MailIntegrationNotificationPlugin=You received {0} emails within the last {1} minutes
+UINotification.messageItems.received.MailIntegrationNotificationPlugin=message received from
```

```suggestion
mailIntegration.notification.drawer.message.received.from=message received from
```
Move that label to the part "mail integration settings portlet drawer"